site/content/en/concepts/functions/_index.md

```diff
@@ -49,6 +49,9 @@ See [Configuration Functions Specification][spec] for further details.
 
 There are two special-case functions: source functions and sink functions.
 
+Note: Source functions and sink functions are *functions*. They are invoked by command `kpt fn run` instead of `kpt fn source` or `kpt fn sink`.
```

Comment:
Can you split this note into two? One note for source functions which goes at the bottom of the Source Function section and other note at bottom of Sink Function section. I think it's better to introduce the concept first and then give the clarification.

And let's add a question to the FAQ about this https://googlecontainertools.github.io/kpt/faq/